modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,185 @@
 
 Query is a service that retrieves traces from storage and hosts the user interface to display them.
 
-.Parameters used by the {TempoOperator} to define Query
+.Configuration parameters for the Querier
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    replicas:
-|Specifies the number of Query replicas to create.
-|Positive integer
+|`nodeSelector`
+|NodeSelector is the simplest recommended form of node selection constraint.
+|type: object, additionalProperties: type: string
 |
+
+|`replicas`
+| Replicas represents the number of replicas to create for this component.
+| type: integer, format: int32
+|
+
+|`tolerations`
+|Tolerations defines component specific pod tolerations.
+|type: array, x-kubernetes-list-type: atomic
+|
+
+|`Tolerations` (items)
+|The pod this Toleration is attached to tolerates any taint that matches the triple <key, value, effect> using the matching operator <operator>.
+|type: object
+|
+
+|`effect`
```

Comment:
The `querier` component has a field `tolerations`, which contains a list of `Toleration`, which have `effect` etc.: https://github.com/grafana/tempo-operator/pull/711/files#diff-e7108bb7c82881b86a832ed14cefa0297fc0d94c44f4ea0029762334e68fe272R139-R148

With the current style it looks like `effect` is a property on `querier`, but that's wrong. It is a property of the `Toleration` item inside the `tolerations` list.

Reply:
What about https://github.com/openshift/openshift-docs/pull/66373/files#r1441598504?

Reply:
Thank you for clarifying this. Please take a looking at my open suggestions above and below.